sass/variables/_sizing.scss

```diff
@@ -125,3 +125,12 @@ $base-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 10p
 $hover-radio-box-shadow: 0 0 0 1px $alert-primary-active inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary-active inset;
 $checked-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary inset;
 $active-radio-box-shadow: 0 0 0 8px $alert-primary-active inset, 0 0 0 16px $white inset;
+
+// Tables
+$table-margin: $base-margin;
+$table-row-divider-width: 1px;
+$table-header-padding: $base-margin;
```

Comment:
`$base-padding` (24px) too much? I normally use `$base-padding` for the space around content within boxes. If this is per Susan maybe we should consider making padding smaller everywhere else as well

Also, this is duplicated on line 136